script.js

```diff
@@ -1,9 +1,129 @@
 // eslint-disable-next-line no-unused-vars
 function closeMenu() {
   document.getElementById('myButtons').style.display = 'none';
+  const blur = document.getElementById('blur');
+  blur.classList.toggle('active');
 }
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] It is not acceptable to disable any linter rules because you are preventing them from checking any syntax errors which can make your project prone to bugs in the future. **We should not disable linter rules rather we should work to correct them.** 
For this `no-unused-vars` is enforcing declaring variables that you will be using only. if you are not going to need it then you should remove it 😉 This is among coding best practices YAGNI or You Are Not Gonna Need It. Please look back in your codes and see **if there is somewhere you have an initialized variable that you didn't use throughout your code 🏃**

Reply:
the function does not need to be called, but we need it, otherwise the button does not work. several other people have used this solution, because we do not know what else to do. for all of them this was accepted

Reply:
OK. I get it 👍